fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -1154,7 +1179,7 @@ private void assertDataTableEmpty(final String datatableName) {
 
     private int getRowCount(final String datatableName) {
         final String sql = "select count(*) from " + sqlGenerator.escape(datatableName);
-        return this.jdbcTemplate.queryForObject(sql, Integer.class); // NOSONAR
+        return this.jdbcTemplate.queryForObject(sql, Integer.class);
```

Comment:
Please don't remove the NOSONAR note from here.

Reply:
Done!